db/migrate/20130908170524_add_keys.rb

```diff
@@ -3,8 +3,6 @@ def change
     # turn off Foreign Key checks
     if ActiveRecord::Base.connection.adapter_name == 'PostgreSQL'
```

Comment:
Is this normalized? Also, other places use `ActiveRecord::Base.connection.instance_values['config'][:adapter]`. What's the best way?

Reply:
I think `adapter_name` is considered the correct way, but was maybe not available reliably in the early days (which is why the `instance_values` is used mostly in the older migrations). In any case I wanted to minimize the code changes here so didn't change it. We could do a refactor in separate PR to normalize it everywhere.

Reply:
I'd very much be in favor of squashing all. Given all the deprecations I suspect this will be a post-2.0 thing. Perhaps in a year we should do a 3.0 that drops all migrations prior to 2.0